.github/prep-hotfix-sources.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -x
+
+HOTFIX_TAG=$1
+REPO_PATH=$2
+SPEC_PATH=$3
+
+# Create OSG build dirs
+build_dir=$(pwd)/_build_dir
+for dirname in osg upstream; do
+    mkdir -p $build_dir/$dirname
+done
+
+version=$(echo "$HOTFIX_TAG" | sed "s/v\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/")
+# Remove illegal dashes in release string
+release=$(tr '-' '.' <<< 1.${HOTFIX_TAG#*-})
```

Comment:
what is the expected format of `HOTFIX_TAG` ?

Reply:
I see now in the description that hotfix tags look like `vX.Y.Z-osghotfix-A`.  So apparently you want that to translate to
`version=vX.Y.Z` and `release=osghotfix.A` .

In which case it looks like it'll work how you have it.

Alternatively, if you want to keep it all in bash, you can do something like
```
[[ $HOTFIX_TAG =~ ^(v[0-9]+\.[0-9]+\.[0-9]+)-(.*) ]]
version=${BASH_REMATCH[1]}
release=${BASH_REMATCH[2]//-/.}
```